chain/jsonrpc/src/lib.rs

```diff
@@ -380,7 +380,7 @@ impl JsonRpcHandler {
             "block" => process_method_call(request, |params| self.block(params)).await,
             "broadcast_tx_async" => {
                 process_method_call(request, |params| async {
-                    let tx = self.send_tx_async(params).await.to_string();
+                    let tx = self.send_tx_async(params).to_string();
```

Comment:
Why isn't await needed here?

Reply:
Because `send_tx_async` is no longer `async`. 😛

Reply:
Change seems fine to me.